### DISCUSSION TITLE
Whether the VAD model supports steaming recognition?

### DISCUSSION THREAD

**Discussion by uloveqian2021** [speechbrain/speechbrain#1650] [Q&A] [answered]
hello, I want to know whether the VAD model supports steaming recognition?

**Comment by mravanelli** [comment, marked as answer]
Not for now. Online/real-time speech processing is a big topic that we would like to address in the near future.

**Reply by Ryu1845** [reply]
Any updates on this ?
Couldn't find an answer right away.
